docs/guides/insar_product_guide.md

```diff
@@ -0,0 +1,304 @@
+
+You may still find the Geographic, Baseline and SBAS searches in Vertex useful for finding reference scenes or picking specific pairs to use when submitting InSAR jobs via the [SDK](https://hyp3-docs.asf.alaska.edu/using/sdk/) or [API](https://hyp3-docs.asf.alaska.edu/using/api/). 
+
+### Considerations for Selecting an InSAR Pair
+When selecting an InSAR pair, observe the following required conditions:
+
+1. Images from an identical orbit direction (either ascending or descending)
+2. Images with identical incidence angles and beam mode
+3. Images with identical resolution and wavelength (usually from the same sensor)
+4. Images with the same viewing geometry (same path and frame)
+5. Images with identical polarizations (both HH or VV)
+
+In addition, the following suggestions may be helpful:
+
+1. Use images from similar seasons/growth/weather conditions
+2. For deformation mapping: limited spatial separation of acquisition locations (small physical baseline)
+3. For topographic mapping: limited time separation between images (small temporal baseline) 
+
+To analyze deformation caused by a single discrete event, such as an earthquake, select images that bracket the event as closely in time as possible. Keeping the window narrowly focused on the time of the event will reduce the impacts of other processes that may mask the signal of the event of interest.
+
+------
+> **Aside: Critical Baseline**
+>
+> Large baselines are better than small for topographic mapping. However, as the baseline increases, coherence decreases. As some point, it is impossible to create an interferogram because of baseline decorrelation. The maximum viable baseline per platform, referred to as the *critical baseline*, is a function of the distance to the ground, the wavelength, and the viewing geometry of the platform. For Sentinel-1 this critical baseline is about 5 km. In practice, if the perpendicular baseline between images is more than 3/4 of the critical baseline, interferogram creation will be problematic due to the level of noise. 
+> 
+> For deformation mapping, it is best to minimize the perpendicular baseline whenever possible, but there may be tradeoffs in terms of finding suitable temporal baselines. In most cases, however, pairs selected for deformation mapping will have perpendicular baselines *much* smaller than the critical baseline.
+
+------ 
+
+### Processing Options 
+
```

Comment:
We need to add in entries for the new options coming available: 
- DEM
- Water mask
- wrapped phase GeoTIFF?

Reply:
The version of ifm_sentinel.py in develop has options:

```  -r RLOOKS, --rlooks RLOOKS                        Number of range looks (def=20)
  -a ALOOKS, --alooks ALOOKS                      Number of azimuth looks (def=4)
  -l                    Create look vector theta and phi files
  -s                   Create line of sight displacement file
  -w                  Create wrapped phase file
  -i                    Create incidence angle map
  -m                  Create and apply water mask```

The option to include the wrapped phase file has been around for a few months now.

Reply:
Addressed in PR #98